neuralgcm/transforms.py

```diff
@@ -130,12 +130,15 @@ def __call__(self, inputs):
     # an error, as expected. This works because tree_map skips `None` values in
     # the first argument, as long as all dictionary keys match.
     result = jax.tree_util.tree_map(
-        lambda x, y, z: (None if x is None else x * z + y),
-        inputs,
-        shifts,
-        scales,
-        is_leaf=lambda x: x is None,
+      lambda x, y, z: None if x is None else (x * z + y),
+      inputs, 
+      shifts, 
+      scales,
+      is_leaf=lambda x: x is None
     )
+
+    # result = jax.tree_util.tree_map(
+    #     lambda x, y, z: x * z + y, inputs, shifts, scales)
     return from_dict_fn(result)
```

Comment:
This looks like it should do the same thing as the existing code, just with different whitespace formatting?